### PR TITLE
Add a subtle grid to our first welcome screen

### PR DESCRIPTION
**Pull request by jonbell-lot23**
**Background**

Our onboarding screens used to be flat and gray, so last week we looked into adding a bit more pizazz. For our first iteration, we re-architected the code to be more consistent and aligned to a dark theme.

For this PR, I'm adding a subtle Tron grid that gives a little more pizazz without being over-the-top. Here's how the welcome screen looks (top left) compared to other onboarding dialogs:

image

Here's a close-up:
image

Loom describing the work:
https://www.loom.com/share/c17b4b9918aa4ba986c31c31e6f2fb9c

Figma for playing with different visual treatments:
https://www.figma.com/file/mAotNZRv6M5X0zHIFKPzFY/Replay-Design-Doc?type=design&node-id=17343-260575&t=69BPNMf7SAD7OFWm-4

Addresses https://linear.app/replay/issue/DES-686/investigate-incorporating-more-pizzaz-into-early-onboarding-screens

Coming soon: [standardising our other onboarding visuals](https://linear.app/replay/issue/DES-685/standardize-our-onboarding-visuals-stage-2) so they're not so bespoke.

### CODE REVIEW

**Review by jasonLaster — COMMENTED** [review]
Looks good to me. We'll probably want to do an opinionated pass where we add more content to these flows, but this is a more polished pass

File: src/ui/components/shared/Onboarding/index.tsx [hunk 1]
```diff
@@ -138,16 +137,16 @@ export function OnboardingButton({
...
+  visualTreatment,
 }: {
   children: React.ReactNode;
-  // For randomizing some background elements as controlled by progress
-  // on the parent component, e.g. circles/bubbles that change on click
-  randomNumber?: number;
   theme?: "dark" | "light";
+  visualTreatment?: string;
 }) {
+  const themeClass = visualTreatment ? styles[visualTreatment] : "";
```
Comment (bvaughn) [inline src/ui/components/shared/Onboarding/index.tsx:146]: This seems to be unused. Remove it?
```suggestion
}) {
```

**Review by bvaughn — COMMENTED** [review]
Regarding `public/images/grid.png` – 307k is still a pretty big chunk of bandwidth to spend on a single image. I guess Replay is a high-bandwidth app overall...maybe it's okay. :)

**Comment by jasonLaster** [comment]
I think it's fine for the time being. We'll move this to a react component down the line.

**Comment by bvaughn** [comment]
I don't understand your response @jasonLaster 😅  I was just saying that we're adding a lot of file size for these largeish PNG images.

File: src/ui/components/shared/Onboarding/index.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/shared/Onboarding/index.tsx:146]: Indeed it is, thanks!

**Comment by jonbell-lot23** [comment]
Hey gang, I'm squashing and merging. Following up:

* Brian asked about the file size
* Jason said it's fine for now
* (I have thoughts on this for another day)
* I removed some outdated code as per Brian's suggestion

**Pull request merged**